check-environment-and-install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+return_code=0
```

Comment:
return code (a Python concept) and exit status are 2 different things

here it's exit_status

Reply:
fixed